.github/workflows/nightly.yml

```diff
@@ -3,15 +3,9 @@ name: 'Nightly'
 on:
   schedule:
     - cron: '* 14 * * *' # 14:00 in UTC is 6am PST
-    branches:
-      - master
```

Comment:
let's bring this back, we still only want to run this on master we just also wanted to run it during PR while deving

Reply:
NVM: 

> You can schedule a workflow to run at specific UTC times using POSIX cron syntax. Scheduled workflows run on the latest commit on the default or base branch. The shortest interval you can run scheduled workflows is once every 5 minutes.

https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#scheduled-events-schedule

We don't need the branch specifier